common/hipaa-password-icons.html

```diff
@@ -0,0 +1,17 @@
+<link rel="import" href="../../iron-icon/iron-icon.html">
```

Comment:
@sricharan123 
You don't need this file in `common`
Pls move it to the root level.

Typically you have a common if there is something that is shared between multiple components. In this case we just have an element and nothing is shared between elements or components inside this code.